WordPress/gutenberg
# Latest Post Block: Refactor settings panel to use ToolsPanel

## What?
Part of: https://github.com/WordPress/gutenberg/issues/67813

Fixes https://github.com/WordPress/gutenberg/issues/67931

## Why?
Latest Post Block code to include ToolsPanel instead of PanelBody.

## How?
Refactored Latest Post Block code to include ToolsPanel instead of PanelBody.

## Testing Instructions

- Login to Dashboard
- Edit any post/ page
- Add Archive Title block.
- Check settings panel of the Latest Post block and verify the change.

## Screenshots or screencast 

**Before**

![before_posts](https://github.com/user-attachments/assets/e7e3a697-7a90-463e-b574-dd60d9a4132b)


**After**

![after_posts](https://github.com/user-attachments/assets/75033f61-8937-4ef9-9aab-47bf4e0bee95)
